divvy/cloudfilebrowser.scss

```diff
@@ -527,36 +525,37 @@ the License.
         content: '\f15c';
         display: inline;
         font-family: 'FontAwesome';
-        font-size: 13px;
+        font-size: 1.5rem;
         margin-right: 8px;
     }
 
-    #services-containers > div.provisioned > div.listTable > div.scrollPanel > ul li.filename:before { color: white }
-
     #services-containers > div.provisioned > div.listTable > div.scrollPanel > ul li.foldername:before { content: '\f07b'; color: burlywood }
     #services-containers > div.provisioned > div.listTable > ul:first-child {
-        /*position: fixed;*/
-        background: linear-gradient(#555, #333);
-        border-left: 1px solid #777;
-        border-right: 1px solid #333;
-        box-shadow: inset 0 16px 2px rgba(255,255,255,.1), 0 -1px 0 rgba(255,255,255, .3), 0 -2px 0 rgba(0,0,0,.4), 0 1px 1px rgba(0,0,0,0.5);
-        color: #fff;
+        background: rgba(255,255,255,.1);
+        color: #888;
         font-weight: bold;
+        padding-top: 10px;
         text-transform: uppercase;
         width: 100%;
     }
 
+    #services-containers > div.provisioned > div.listTable > div.scrollPanel > ul:nth-child(odd) {
+        background: #f9f9f9;
+    }
+
+    #services-containers > div.provisioned > div.listTable > div.scrollPanel > ul:nth-child(even) {
+        background: white;
```

Comment:
Do we use text-based colors within this whole file? If not, please convert this to hex.